lib/assessment.rb

```diff
@@ -13,20 +15,30 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    (5..20).reduce(:+)
   end
 
   # Question 3
   # Write ruby code that returns true if some of the entries
   # in the numbers enumerable are odd
   def some_odd(numbers)
-    # Your code goes here
+    [1, 2, 3, 4, 5, 6, 7, 8, 9, 10].chunk { |n| n.odd?
+      }.each { |odd, array|
+        p [odd, array]
+      }
+
```

Comment:
Not really sure where you were going with this one. Take a look back at the Ruby Array/Enumerable methods and try this one again.